katello.dot

```diff
@@ -27,27 +27,27 @@ node "server" {
     [Foreman Discovery] as foreman_discovery
     [Foreman SCAP] as foreman_scap
 
-    foreman_tasks -down-> dynflow
-    katello -> foreman
-    katello -> foreman_tasks
-    katello -> foreman_docker
+    foreman_tasks -down..> dynflow
+    katello ..> foreman
+    katello ..> foreman_tasks
+    katello ..> foreman_docker
```

Comment:
I was trying to leave `--->` as dependency requirements for all the various plugins. There are other arrow shapes provided by plantuml that I could try and use as well if you think one of those would be better at indicating this: http://plantuml.com/sequence.html#Arrow